package.json

```diff
@@ -30,7 +30,7 @@
   ],
   "dependencies": {
     "@octokit/rest": "^18.0.6",
-    "babel-runtime": "^6.26.0",
+    "babel-runtime": "^7.12.5",
```

Comment:
[`babel-runtime`](https://www.npmjs.com/package/babel-runtime) is old. This should be [`@babel/runtime`](https://www.npmjs.com/package/@babel/runtime).